source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -546,21 +546,25 @@ def _tag_by_mode(responses):
         for j in r.journeys:
             _tag_journey_by_mode(j)
 
-def _switch_back_to_ridesharing(response):
-    for journey in response.journeys:
-        for i, section in enumerate(journey.sections):
-            if section.type == response_pb2.STREET_NETWORK \
-                    and section.street_network.mode == response_pb2.Car:
-                if (len(journey.sections) > i+1 and journey.sections[i+1].type != response_pb2.PARK) or \
-                        (i > 0 and journey.sections[i-1].type != response_pb2.LEAVE_PARKING) \
-                        or len(journey.sections) == 1:
-                    #TODO: handle crowfly :(
-                    section.street_network.mode = response_pb2.Ridesharing
-                    journey.durations.ridesharing += section.duration
-                    journey.durations.car -= section.duration
-                    journey.distances.ridesharing += section.length
-                    journey.distances.car -= section.length
 
+def _is_fake_car_section(section):
+    """
+    This function test if the section is a fake car section
+    """
+    return (section.type == response_pb2.STREET_NETWORK or section.type == response_pb2.CROW_FLY) and \
+           section.street_network.mode == response_pb2.Car
```

Comment:
If you do a journey with `first_section_mode[]=car&first_section_mode[]=ridesharing&first_section_mode[]=walking` it will not work.

Reply:
@TeXitoi then, it's impossible to achieve this PR's objective in a simple way... since when the OSM is missing, i can't tell if the journey is for car or ridesharing....

Reply:
At that point we only have one dep_mode and one arr_mode (already dispatched, and tested earlier), no?
To be tested but it looks OK.

Reply:
If a test show that it's working, OK.